### PR TITLE
net-analyzer/speedtest-cli - add setuptools dependency (bug# 502302)

### PR DESCRIPTION
**Pull request by mm1ke**
This annoyed me for quite some time. Especially on minimal systems, were setuptools wasn't pulled by any other packages, i've hit this quite often. Would be nice if we could finally add this dependency.

Gentoo-Bug: https://bugs.gentoo.org/show_bug.cgi?id=502302
Btw, the bug is for Version 0.2.4, but it's still valid for 0.2.5 - just testet it + see duplicate.

Maybe this should also block: https://bugs.gentoo.org/show_bug.cgi?id=530994

### CODE REVIEW

File: net-analyzer/speedtest-cli/speedtest-cli-0.2.5.ebuild [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2015 Gentoo Foundation
+# Copyright 1998-2015 Gentoo Foundation
```
Comment (mgorny) [inline net-analyzer/speedtest-cli/speedtest-cli-0.2.5.ebuild]: Please don't move copyright back :).

**Comment by mgorny** [comment]
Please fix the copyright year and then I'll merge it.

**Comment by prometheanfire** [comment]
dunno if we have this set up

**Comment by prometheanfire** [comment]
test this please

**Comment by mgorny** [comment]
Merged.

Now a quick extra lesson: in trivial-yet-important fixes like this, you may consider adding them to older stable version too. I just did that after confirming it uses setuptools as way. While you generally should be extra cautious when dealing with stable ebuilds, adding extra `DEPEND` to unbreak them is rather fine :).

**Pull request closed without merging**